### DISCUSSION TITLE
How can I connect to multiplayer servers from MCPElauncher?

### DISCUSSION THREAD

**Discussion by flashgnash** [minecraft-linux/mcpelauncher-manifest#1035] [Q&A]
I assume this was not previously the case but it seems like now the minecraft pocket edition client requires a sign in to a microsoft account to join any multiplayer server at all and as far as I've heard there's still not a mechanism to sign into an xbox account using this launcher

**Comment by Gabriel-Science** [comment]
> [...] there's still not a mechanism to sign into an xbox account using this launcher
> 
> 

For me it worked just fine, it should be your computer who doesn't support another window opening...
Sorry.
